libs/parallelism/algorithms/tests/unit/container_algorithms/remove_copy_if_range.cpp

```diff
@@ -16,9 +16,70 @@
 #include <string>
 #include <vector>
 
+#include "iter_sent.hpp"
 #include "test_utils.hpp"
 
 ////////////////////////////////////////////////////////////////////////////
+// This test should be valid once remove_adapt is accepted and
+// iter_sent implements a valid sentinel
+
+// template <typename ExPolicy>
+// void test_remove_copy_if_sent(ExPolicy policy)
+// {
+//     static_assert(hpx::is_execution_policy<ExPolicy>::value,
+//         "hpx::is_execution_policy<ExPolicy>::value");
+
+//     using hpx::get;
+
+//     std::size_t const size = 100;
+//     std::vector<std::int16_t> c(size), d(size);
+//     std::iota(std::begin(c), std::end(c), 1);
+
+//     auto pred = [](std::int16_t const& a) -> bool { return a % 42 == 0; };
+
+//     auto pre_result = std::count_if(std::begin(c), std::end(c), pred);
+//     hpx::ranges::remove_copy_if(
+//         std::begin(c), sentinel<std::int16_t>{50}, std::begin(d), pred);
+//     auto post_result = std::count_if(std::begin(c), std::end(c), pred);
+
+//     HPX_TEST(pre_result == 2 && post_result == 1);
+// }
```

Comment:
Can this comment be removed?

Reply:
This is a valid test that needs the recent changes on our custom sentinel (iter_sent.hpp) in order to run successfuly. I could either rebase on the unmerged branch (maybe?) or wait for #5125 to get merged.

Reply:
@gonidelis #5125 has been merged now, could you update this, please?

Reply:
@hkaiser  already have... sorry for not putting this as resolved ;)